plan-release-template.yml.ejs

```diff
@@ -53,24 +53,18 @@ jobs:
           ref: '<%= defaultBranch %>'
       - uses: actions/setup-node@v4
         with:
-          node-version: 18
-      <% if (pnpm) { %>
-      - uses: pnpm/action-setup@v3
+          node-version: 18<% if (pnpm) { %>
+      - uses: pnpm/action-setup@v4<% if (!hasPackageManager) { %>
         with:
```

Comment:
why specify the version at all? the duplication in workflows is maddening. 

This is what package.json#packageManager is for

Reply:
or wait, I can't read ejs.
this does exactly what I expect.

Reply:
look a bit more carefully at the change, it only adds the version if you haven't set packageManager. And version is a required parameter if you haven't got packageManager so the action will break without it

I agree that packageManager should be used by most people at this stage but we can't impose it as part of our workflow. And people who do define it will benefit from it 💪

Reply:
I think we crossed paths 
![image](https://github.com/user-attachments/assets/50706fae-4565-4587-9d36-ae9e98f5e36d)

Reply:
omg Github 😭 this is why eventual consistency doesn't always work. it didn't even update the thread **after** I posted my response